ideas/cosmwasm-chain.md

```diff
@@ -0,0 +1,15 @@
+# CosmWasm Chain
+
+### A blockchain for the [CosmWasm](github.com/cosmwasm) module with smart contracts generated in Rust
+
+## Description:
+This blockchain could be the testnet first proposed by [ethanfrey](https://github.com/ethanfrey) in his [blog article](https://github.com/cosmwasm/docs/blob/update-blog/website/blog/2019-11-25-cosmwasm-intro.md) describing CosmWasm. This would be a stable playground for users to try out contract interactions. 
```

Comment:
```suggestion
This blockchain could be the testnet first proposed by [ethanfrey](https://github.com/ethanfrey) in his [blog article](https://www.cosmwasm.com/blog/2019/12/11/cosmwasm-intro) describing CosmWasm. This would be a stable playground for users to try out contract interactions. 
```